src/widgets/school-menu/ui/school-menu/school-menu.test.tsx

```diff
@@ -0,0 +1,143 @@
+import { screen } from '@testing-library/react';
+import { SchoolMenu } from '../school-menu/school-menu';
+import { Course } from '@/entities/course';
+import { mockedCourses } from '@/shared/__tests__/constants';
+import { renderWithRouter } from '@/shared/__tests__/utils';
+import { COURSE_TITLES, schoolMenuStaticLinks } from 'data';
+
+describe('SchoolMenu', () => {
+  const aws = mockedCourses.find(
+    (course) => course.title === COURSE_TITLES.AWS_FUNDAMENTALS,
+  ) as Course;
+  const react = mockedCourses.find((course) => course.title === COURSE_TITLES.REACT) as Course;
+
+  it('renders without crashing and displays "rs school" heading', () => {
+    renderWithRouter(
+      <SchoolMenu heading="rs school">
+        {schoolMenuStaticLinks.map((link) => (
+          <SchoolMenu.Item
+            key={link.title}
+            title={link.title}
+            description={link.description}
+            url={link.detailsUrl}
+          />
+        ))}
+      </SchoolMenu>,
+    );
+
+    const headingElement = screen.getByRole('heading', { name: /rs school/i });
+
+    expect(headingElement).toBeInTheDocument();
+  });
+
+  it('displays correct links and descriptions with "rs school" props', () => {
+    const { container } = renderWithRouter(
+      <SchoolMenu heading="rs school">
+        {schoolMenuStaticLinks.map((link) => (
+          <SchoolMenu.Item
+            key={link.title}
+            title={link.title}
+            description={link.description}
+            url={link.detailsUrl}
+          />
+        ))}
+      </SchoolMenu>,
+    );
+
+    expect(screen.getAllByRole('link')).toHaveLength(2);
+
+    const links = screen.getAllByRole('link');
+
+    links.forEach((link) => {
+      expect(link).toBeInTheDocument();
+    });
+
+    const descriptions = container.getElementsByTagName('small');
+
+    for (const description of descriptions) {
+      expect(description).toBeInTheDocument();
+    }
```

Comment:
Can we use the similar assertion to `expect(screen.getAllByRole('link')).toHaveLength(2);` here?

Reply:
fixed